src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -214,10 +214,32 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.clickEditSessionButton();
         editedSession.setInstructions(new Text("Made some changes"));
         feedbackEditPage.editFeedbackSession(editedSession.getEndTime(), editedSession.getStartTime(),
-                                        editedSession.getInstructions(), editedSession.getGracePeriod());
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), true);
 
         String expectedString = "The end time for this feedback session cannot be earlier than the start time.";
         feedbackEditPage.verifyStatus(expectedString);
+
+        ______TS("test discard changes to feedback session");
+
+        // Make changes to Feedback session
+        editedSession.setStartTime(Const.TIME_REPRESENTS_NOW);
+        editedSession.setEndTime(Const.TIME_REPRESENTS_LATER);
+        editedSession.setInstructions(new Text("Made some more changes"));
+        editedSession.setGracePeriod(15);
+        feedbackEditPage.editFeedbackSession(editedSession.getStartTime(), editedSession.getEndTime(),
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), false);
```

Comment:
While adding this test, a bug in the feature came to light. Following steps:
- Click edit button
- Make some changes to the FS details
- Click change uncommon email settings link
- Click discard changes --> OK
- Only the changes made to the Feedback Session Instructions field are not discarded.

I can confirm that the changes to FS Instructions field ARE discarded if I do not click the `change uncommon email settings` button. I can also confirm that the same error occurs if I click the `change uncommon visibility options` as well. So there seems to be some problem with how the JS handles these two buttons. Am looking into this.